src/python/pants/backend/native/subsystems/native_toolchain.py

```diff
@@ -6,9 +6,8 @@
                         unicode_literals, with_statement)
 
 from pants.backend.native.config.environment import (Assembler, CCompiler, CppCompiler,
-                                                     GCCCCompiler, GCCCppCompiler,
-                                                     Linker, LLVMCCompiler, LLVMCppCompiler,
-                                                     Platform)
+                                                     GCCCCompiler, GCCCppCompiler, Linker,
```

Comment:
nit: Is it okay to submit formatting changes along with features in a single PR?

Reply:
yeah, this is just a side effect of running `./build-support/bin/isort.sh -f` against a dirty master.